spec/models/app_configuration_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe AdminOnly::AppConfiguration, type: :model do
+  let(:app_configuration) { create(:app_configuration) }
+
+  describe 'Factory' do
+    it 'has a valid factory' do
+      expect(create(:app_configuration)).to be_valid
+    end
+  end
+
+  describe 'DB Table' do
+    it { is_expected.to have_db_column :id }
+    it { is_expected.to have_db_column :chair_signature_file_name }
+    it { is_expected.to have_db_column :chair_signature_content_type }
+    it { is_expected.to have_db_column :chair_signature_file_size }
+    it { is_expected.to have_db_column :chair_signature_updated_at }
+    it { is_expected.to have_db_column :shf_logo_file_name }
+    it { is_expected.to have_db_column :shf_logo_content_type }
+    it { is_expected.to have_db_column :shf_logo_file_size }
+    it { is_expected.to have_db_column :shf_logo_updated_at }
+  end
+
+  describe 'Validations' do
+    it do
+      is_expected.to validate_attachment_content_type(:chair_signature)
+        .allowing('image/png', 'image/jpeg')
+        .rejecting('image/gif', 'image/bmp')
+    end
+    it do
+      is_expected.to validate_attachment_content_type(:shf_logo)
+        .allowing('image/png', 'image/jpeg')
+        .rejecting('image/gif', 'image/bmp')
+    end
+
+    describe 'validates file contents and file type - chairperson signature' do
+      let(:file_root) { "#{Rails.root}/spec/fixtures/member_photos/" }
```

Comment:
It'd be appropriate to set a constant for the file fixtures and just use that in the specs/tests.  You're not setting some complicated constellation of data with it; the constant  is clear about what the intent is. 

Ex:  
```
MEMBER_PHOTOS_PATH = File.join(Rails.root, 'spec', 'fixtures', 'member_photos')

let(:txt_file)  { File.new( File.join(MEMBER_PHOTOS_PATH, 'text_file.jpg') ) }
```
(Should use `File.join` instead of hard coding the "/" path separator.)


More generally but outside the scope of this PR, we should correct the path set in `rails_helper.rb` because currently it is defined specifically for just one area of specifications (`config.file_fixture_path = 'spec/fixtures/uploaded_files'`).

  If we remove that line so that it uses the default fixtures directory, then we can just narrow the fixture path as appropriate within each specification and use the `file_fixture` method.  Ex:
 ```
MEMBER_PHOTOS_ROOT = '/member_photos'
let(:txt_file)  { file_fixture( File.join(MEMBER_PHOTOS_ROOT, 'text_file.jpg') ) }
```

or even:
```
let(:txt_file)  { file_fixture( File.join('member_photos', 'text_file.jpg') ) }
```

_This is a chore that we need to put in to PT if everyone agrees._

Reply:
Yes, I agree with that idea.